packages/beacon-node/test/sim/6110-interop.test.ts

```diff
@@ -139,29 +143,28 @@ describe("executionEngine / ExecutionEngineHttp", function () {
     const newPayload = {
-      parentHash: dataToBytes("0x26118cf71453320edcebbc4ebb34af5b578087a32385b80108bf691fa23efc42", 32),
-      feeRecipient: dataToBytes("0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b", 20),
-      stateRoot: dataToBytes("0x14208ac0e218167936e220b72d5d5887a963cb858ea2f2d268518f014a3da3fa", 32),
-      logsBloom: dataToBytes(
-        "0x10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008000000000000000000000000",
-        256
+      parentHash: fromHexString("0x26118cf71453320edcebbc4ebb34af5b578087a32385b80108bf691fa23efc42"),
+      feeRecipient: fromHexString("0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"),
+      stateRoot: fromHexString("0x14208ac0e218167936e220b72d5d5887a963cb858ea2f2d268518f014a3da3fa"),
+      logsBloom: fromHexString(
+        "0x10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008000000000000000000000000"
       ),
-      prevRandao: dataToBytes("0x0000000000000000000000000000000000000000000000000000000000000000", 32),
+      prevRandao: fromHexString("0x0000000000000000000000000000000000000000000000000000000000000000"),
       gasLimit: 30000000,
       gasUsed: 84846,
       timestamp: 16,
-      extraData: dataToBytes("0x", 0),
+      extraData: fromHexString("0x"),
       baseFeePerGas: 7n,
       excessBlobGas: 0n,
-      transactions: [dataToBytes(depositTransactionA, null)],
+      transactions: [fromHexString(depositTransactionA)],
       withdrawals: [],
       depositReceipts: [depositReceiptA],
       blockNumber: 1,
-      blockHash: dataToBytes(newPayloadBlockHash, 32),
-      receiptsRoot: dataToBytes("0x79ee3424eb720a3ad4b1c5a372bb8160580cbe4d893778660f34213c685627a9", 32),
+      blockHash: fromHexString(newPayloadBlockHash),
+      receiptsRoot: fromHexString("0x79ee3424eb720a3ad4b1c5a372bb8160580cbe4d893778660f34213c685627a9"),
       blobGasUsed: 0n,
     };
-    const parentBeaconBlockRoot = dataToBytes("0x0000000000000000000000000000000000000000000000000000000000000000", 32);
+    const parentBeaconBlockRoot = fromHexString("0x0000000000000000000000000000000000000000000000000000000000000000");
```

Comment:
what's the reason to replace the dataToBytes calls? `data` here refers to the denomination in engine API lingo of execution engine types, which is a 0x prefixed binary data.